server/graphql/v1/mutations/orders.js

```diff
@@ -29,7 +32,7 @@ const oneHourInSeconds = 60 * 60;
 
 const debug = debugLib('orders');
 
-async function checkOrdersLimit(order, remoteUser, reqIp) {
+async function checkOrdersLimit(order, reqIp) {
   if (['ci', 'test'].includes(config.env)) {
```

Comment:
I know this isn't part of your PR but I just noticed this and didn't realise we could include special return cases for CI! 🤯